src/components/TestToolMenu.tsx

```diff
@@ -93,6 +96,36 @@ function TestToolMenu({user = USER_DEFAULT, network}: TestToolMenuProps) {
                 />
             </TestToolRow>
 
+            <TestToolRow title={translate('initialSettingsPage.troubleshoot.exportToFile')}>
+                <Button
+                    small
+                    text={translate('common.export')}
+                    onPress={() => Troubleshooting.exportOnyxDataToFile()}
```

Comment:
Out of curiosity, why do you suggest that? Looking at all the other components in this screen, their `onPress` callbacks are defined inline.

Reply:
moved to a `useCallback` defined before the return.

Reply:
I think this is a holdover from some of our old style guides. I think it's good because:

1. It makes the `return` statement more straightforward and simple because it's only containing render logic
2. It puts all the callbacks in a consistent place

I went through our style guide and the airbnb style guide and I couldn't find this rule anywhere, so I guess it's NAB.